### PR TITLE
feat(shopify): add table with list data and state

### PR DESCRIPTION
**Pull request by cbernat**
- Add table with list data:

image

- Add synchonizing state

image

- Updated UI library version

### CODE REVIEW

File: src/i18n/en.js [hunk 1]
```diff
@@ -185,7 +186,7 @@ export default {
     list_subtitle: `You can synchronize the data manually whenever you want.`,
     list_title: `Synchronized List`,
     table_list: `List Name`,
-    table_subscribers_count: `Subscribers`,
+    table_subscribers_count: `Shopify customers`,
```
Comment (andresmoschini) [inline src/i18n/en.js]: What do you think about also changing the key? maybe:

```suggestion
    table_shopify_customers_count: `Shopify customers`,
```

File: src/components/Integrations/Shopify/Shopify.js [hunk 2]
```diff
@@ -39,6 +41,33 @@ const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
...
+          <tr>
+            <td>{list.name}</td>
+            <td>{list.amountSubscribers}</td>
+          </tr>
+        ) : (
+          <tr className="sync">
+            <td>{list.name}</td>
+            <td className="text-sync">
+              <span className="ms-icon icon-clock"></span>
+              Synchronizing...
```
Comment (andresmoschini) [inline src/components/Integrations/Shopify/Shopify.js]: This is new, right? what do you think about adding a test?

File: src/i18n/en.js (hunk 1 above)
Comment (cbernat) [reply src/i18n/en.js]: I thought that maybe in the future we will reuse this as subscribers, but ... YAGNI.

File: src/components/Integrations/Shopify/Shopify.js (hunk 2 above)
Comment (cbernat) [reply src/components/Integrations/Shopify/Shopify.js]: Also I've missed the resourced text here. Sure a test can help.

**Pull request merged**